playbooks/02 - Use Case - CICD/Pull Latest Content from Source Control.json

```diff
@@ -254,12 +254,41 @@
                 "step_variables": []
             },
             "status": null,
-            "top": "1940",
+            "top": "2180",
             "left": "120",
             "stepType": "\/api\/3\/workflow_step_types\/2597053c-e718-44b4-8394-4d40fe26d357",
             "group": null,
             "uuid": "f183e6b1-7bc4-4fa1-bbe5-c78d383d64b0"
         },
+        {
+            "@type": "WorkflowStep",
+            "name": "Delete Pending Manual Inputs",
+            "description": null,
+            "arguments": {
+                "when": "{{vars.steps.Get_Pending_Manual_Inputs.data['hydra:member'] | length > 0}}",
+                "params": {
+                    "iri": "\/api\/wf\/api\/manual-wf-input\/{{vars.item.id}}\/?format=json",
+                    "body": "",
+                    "method": "DELETE"
+                },
+                "version": "3.6.0",
+                "for_each": {
+                    "item": "{{vars.steps.Get_Pending_Manual_Inputs.data['hydra:member']}}",
```

Comment:
will it delete all manual inputs or only related to CICD?

Reply:
Only Related Change Request